scripts/qa-vdb

```diff
@@ -67,6 +67,10 @@ usage <<-EOU
 	Missing slots won't be displayed when a binding operator (:=) is used as
 	portage records the slot even if it was missing from the ebuild.
 
+    Installed packages' files are considered as candidates only.
+
+    There won't be a warning if a library isn't provided by any package.
```

Comment:
Indenting here needs to be tabs, or it won't be aligned with the above block in output. Given this is about 1 same limitation I wouldn't put a blank line in between too.

imo wouldn't hurt to remind that portage can warn too, maybe do this?
```
	No warning is given in the event libraries are missing entirely or not
	associated with a package. For the former, please use portage's default
	enabled FEATURES="qa-unresolved-soname-deps" to be informed.
```

Reply:
`in` == `if`?

Reply:
No? Could use `if` if did "No warning is given if libraries are missing entirely ..", but "if the event" doesn't work without `in`. Not that I'm a reference for using English properly.

Also, the previous line about installed files is still there (seems redundant?).

Reply:
Ok, my English is not perfect, sorry :)